apps/web/src/shared/apis.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  createAuthClient,
+  createBareClient,
+  getApis,
+  GoogleAuth,
+} from '@affine/datacenter';
+
+import { config } from './env';
+
+let prefixUrl = '/';
+if (typeof window === 'undefined') {
+  // SSR
+  if (config.serverAPI.startsWith('100')) {
+    // This is for Server side rendering support
+    prefixUrl = new URL('http://' + config.serverAPI + '/').origin;
```

Comment:
Can the protocol here be `https`?

Reply:
it's http. Anyway this field haven't been used really so it wont make things danger